frontend/src/scenes/pipeline/Pipeline.tsx

```diff
@@ -32,6 +33,7 @@ export function Pipeline(): JSX.Element {
             />
 
             {!currentTab ? <Spinner /> : currentTab === PipelineTabs.Transformations ? <Transformations /> : null}
+            {currentTab === PipelineTabs.AppsManagement && <AppsManagement />}
```

Comment:
If we're doing a `currentTab`-based conditional expression, let's just do one instead of multiple - these can be combined